frontend/src/app/components/UpdateList/index.scss

```diff
@@ -118,10 +118,12 @@
 
     &:hover {
       border-color: $transparent-white-2;
+      background:rgba(255,255,255,0.05);
```

Comment:
You can use the sass variable $transparent-white-05 here (https://github.com/mozilla/testpilot/blob/master/frontend/src/styles/_utils.scss)